internal/fusekernel/fuse_kernel.go

```diff
@@ -714,12 +716,16 @@ type InitIn struct {
 const InitInSize = int(unsafe.Sizeof(InitIn{}))
 
 type InitOut struct {
-	Major        uint32
-	Minor        uint32
-	MaxReadahead uint32
-	Flags        uint32
-	Unused       uint32
-	MaxWrite     uint32
+	Major               uint32
+	Minor               uint32
+	MaxReadahead        uint32
+	Flags               uint32
+	MaxBackground       uint16
+	CongestionThreshold uint16
+	MaxWrite            uint32
+	TimeGran            uint32
+	MaxPages            uint16
```

Comment:
After MaxPages, it looks like you’re missing `MapAlignment uint16`? https://github.com/torvalds/linux/blame/c500bee1c5b2f1d59b1081ac879d73268ab0ff17/include/uapi/linux/fuse.h#L754

Reply:
Fixed, thanks.